common_util/src/avro.rs

```diff
@@ -166,3 +241,19 @@ fn may_union(val: Value, is_nullable: bool) -> Value {
         val
     }
 }
+
+fn parse_one_row(schema: &AvroSchema, mut raw: &[u8], row: &mut Vec<Datum>) -> Result<()> {
```

Comment:
The tests for newly added methods seems missing, and I guess they should and need these unit tests.

Reply:
Create an issue to track this. #516